docs/spec.md

```diff
@@ -572,25 +572,26 @@ tensor. Depending on the element type, does the following:
 * For signed integers: integer modulus.
 * For floats: `abs` from IEEE-754.
 * For complex numbers: complex modulus.
+* For quantized types: `dequantize_op_quantize(abs, operand, type(result))`.
 
 #### Inputs
 
-| Label | Name      | Type                                                      | Constraints |
-|-------|-----------|-----------------------------------------------------------|-------------|
-| (I1)  | `operand` | tensor of signed integer, floating-point, or complex type | (C1), (C2)  |
+| Label | Name      | Type                                                                                     | Constraints |
+|-------|-----------|------------------------------------------------------------------------------------------|-------------|
+| (I1)  | `operand` | tensor of signed integer, floating-point, or complex type or per-tensor quantized tensor | (C1-C2)     |
 
 #### Outputs
 
-| Name     | Type                                            | Constraints |
-|----------|-------------------------------------------------|-------------|
-| `result` | tensor of signed integer or floating-point type | (C1), (C2)  |
+| Name     | Type                                                                           | Constraints |
+|----------|--------------------------------------------------------------------------------|-------------|
+| `result` | tensor of signed integer or floating-point type or per-tensor quantized tensor | (C1-C2)     |
 
 #### Constraints
 
-* (C1) `shape(result) = shape(operand)`.
-* (C2) `element_type(result)` is defined as:
+* (C1) `shape(operand) = shape(result)`.
```

Comment:
We probably don't need the change to C1, to keep the PR focused?

Reply:
done.